apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -634,6 +635,7 @@ export const pageRouter = router({
                   // follow the same format as autogenerated index pages
                   summary: `Pages in ${parent.title}`,
                 },
+                fallbackIfNoImage: false,
```

Comment:
issue: hmm we added this here but this key is not part of the `index` layout, should it be added?